digdag-core/src/main/java/io/digdag/core/log/LocalFileLogServerFactory.java

```diff
@@ -207,13 +215,19 @@ public void log(byte[] data, int off, int len)
             public void close()
             {
                 try {
-                    temporaryOutput.close();
-
-                    try (InputStream in = Files.newInputStream(temporaryPath);
-                         OutputStream conclusiveOutput = new GZIPOutputStream(Files.newOutputStream(conclusivePath, CREATE, APPEND), 16*1024)) {
-                        ByteStreams.copy(in, conclusiveOutput);
-                        conclusiveOutput.close();
-                        temporaryPath.toFile().delete();
+                    output.close();
```

Comment:
What if this call throws an exception? The temporary file will be remained forever?

Reply:
Yes remained, should we retrying deletion?